### PR TITLE
Update appearance of buttons for password and secretTextarea matching 'jenkins-button's

### PR DESCRIPTION
**Pull request by mawinter69**
replace old style buttons with new jenkins-button

Before: 
image
image

After:
image
image

### Testing done
Manual testing
create password credential and update it
create ssh username with private key credential and update it
Verified that things still work as expected

### Proposed changelog entries

- Update appearance of buttons for password and secretTextarea matching 'jenkins-button's.

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/8179"></a>

### CODE REVIEW

**Comment by mawinter69** [comment]
The plugin https://plugins.jenkins.io/hp-application-automation-tools-plugin/ has a selector on the input button here:
https://github.com/jenkinsci/hpe-application-automation-tools-plugin/blob/425f167f1f78b8f80adab875150dceebdd8d1044/src/main/webapp/js/almUtils.js#L75

**Review by Wadeck — COMMENTED** [review]
👍 Fine security wise

**Review by NotMyFault — APPROVED** [review]
Thanks!

**Review by timja — APPROVED** [review]

**Comment by NotMyFault** [comment]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Review by wasup-yash — APPROVED** [review]

**Comment by MarkEWaite** [comment]
A minor test change was needed in the docker-commons plugin as a result of the change from `textarea` to `input` for the secret text field.  Change is in:

* https://github.com/jenkinsci/docker-commons-plugin/pull/140

Other plugins that use HTMLUnit to create credentials may need a similar change.

I'm a little perplexed that [SecretTextAreaTest](https://github.com/jenkinsci/jenkins/blob/jenkins-2.413/test/src/test/java/lib/form/SecretTextareaTest.java#L110) in Jenkins core is not affected by the same change, but I assume it is not affected because the secret button that is being created in that test is using  the old secrets button instead of the newer and better looking secrets button.

@mawinter69 does that make sense to you, or have I misunderstood something?

**Comment by mawinter69** [comment]
The change in core also changed in the test the way the button is selected and clicked. The same problem is in docker-commons. The button is not clicked (it looks for an input but is now a button) that is why the content is still in an input and not a textarea.

**Pull request merged**
